### PR TITLE
Support es modules

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/svg/svgo/issues/1579

In config of course. Projects with type:module can now use modules to
export config

```js
export default {
  plugins: []
}
```

Also added support for resolving svgo.config.mjs and svgo.config.cjs.

Moved loadConfig tests to svgo-node tests.

mjs test is skipped for now in node 10, just don't use modules there

cc @ai

### CODE REVIEW

**Comment by TrySound** [comment]
Wow, worked!

File: lib/svgo-node.js [hunk 1]
```diff
@@ -13,7 +14,25 @@ exports.extendDefaultPlugins = extendDefaultPlugins;
 exports.createContentItem = createContentItem;
 
 const importConfig = async (configFile) => {
-  const config = require(configFile);
+  let config;
+  try {
+    // dynamic import expects file url instead of path and may fail
+    // when windows path is provided
+    const { default: imported } = await import(pathToFileURL(configFile));
```
Comment (matheus1lva) [inline lib/svgo-node.js:21]: Just a comment here: ESM can be imported using dynamic import without any more work to get it working?

**Review by matheus1lva — APPROVED** [review]

File: lib/svgo-node.js (hunk 1 above)
Comment (TrySound) [reply lib/svgo-node.js:21]: With current lts yes. But we still supported node 10 until next major release so we fallback to require in case of error.

**Review by omgovich — APPROVED** [review]
Looks great to me!

Smart! 👨‍🔬 I haven't considered this kind of fallback.

**Pull request merged**